### DISCUSSION TITLE
Vsync/Screen Tearing XFCE

### DISCUSSION THREAD

**Discussion by BraisGz** [dnschneid/crouton#4766] [Q&A]
So... first of all... **Thanks for this amazing tool**... Nothing against ChromeOs... but it is absolutely too young. I managed to get Ubuntu 22.04 LTS with XFCE running just fine but i'm noticing some minor screen tearing or maybe weird framerates i'm not really shure. Is there any way to verify if Vsync is On? 

Im also getting retained packages (  xserver-xorg-core xserver-xorg-input-wacom xserver-xorg-video-intel ) could that become a problem in the future?

**Comment by StupidExperimenter** [comment]
Maybe your computer is struggling to run 22.04 lower your version a bit that could help a lot. That helped me with Debian.
I do recommend Debian instead of Ubuntu since ChromeOS is based on Debian. In conclusion Debian runs better on Chromebooks

**Reply by TomTravis** [reply]
the X display driver is not configured correctly this a maze of crouton command and scripts. The lasted X11 server works however xinit via /usr/local/bin crouton is all hard coded point to the X11 /usr/lib/xinit to start X11 the display drive config is bad the fake X86 drive  fails this is why the graphics and bad the current X11 server runs on Ubuntu and bullseye so the same thing happens on arm64,  x86_x64 is fine. so work is needs

**Comment by quickreactor** [comment]
try switching to compton compositor, that fixed for me

**Comment by ghost** [comment]
*no compton needed*
**use this commands:**


```
xfwm4 --replace --vblank=glx &
xfconf-query -c xfwm4 -p /general/vblank_mode -s glx
```
if you have other screen issues try this too:

```
sudo apt install nano
cd /etc/X11/
sudo mkdir xorg.conf.d && cd xorg.conf.d
sudo nano 20-optimus.conf
```

on nano put this and save: 

```
Section "Screen"
    Identifier "DefaultScreen"
    GPUDevice "Nvidia dGPU"
    Device "Intel iGPU"
EndSection
Section "Device"
    Identifier "Intel iGPU"
    Driver "modesetting"
    BusID "PCI:0:2:0"
EndSection
Section "Device"
    Identifier "Nvidia dGPU"
    Driver "nvidia"
    BusID "PCI:1:0:0"
EndSection
```

**this worked for me :D**
